src/components/Cards/Cards.js

```diff
@@ -0,0 +1,19 @@
+import Card from '../Card/Card';
+
+export default function Cards(props) {
```

Comment:
I recommend you to destructure props. On first sight it seams ugly 😁 -  but It improves the readability of the code. 
```suggestion
export default function Cards({excercises}) {
```